src/test/ceph_objectstore_tool.py

```diff
@@ -44,9 +44,9 @@ def wait_for_health():
     print "Wait for health_ok...",
     tries = 0
     while call("./ceph health 2> /dev/null | grep -v 'HEALTH_OK\|HEALTH_WARN' > /dev/null", shell=True) == 0:
-        if ++tries == 30:
+        if ++tries == 150:
```

Comment:
this is actually a bug. in python "+" is an identity operator. so "++tries" evaluates to "try".